src/test/java/BasePage.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
```

Comment:
Please create a package named 'pages' and move this class into it

![image](https://user-images.githubusercontent.com/104761482/227153790-2af96904-2384-49f2-932b-39ff871161a9.png)

Reply:
Created and refactored classes into pages package.